_includes/footer.html

```diff
@@ -4,7 +4,8 @@
             <ul class="list-inline">
                 <li><a href="https://github.com/namecoin"><i class="fa fa-github fa-2x" title="GitHub">GitHub</i></a></li>
                 <li><a href="https://twitter.com/Namecoin"><i class="fa fa-twitter fa-2x" title="Twitter">Twitter</i></a></li>
-                <li><a href="https://pay.reddit.com/r/namecoin"><i class="fa fa-users fa-2x" title="Reddit">Reddit</i></a></li>
+                <li><a href="https://www.reddit.com/r/namecoin"><i class="fa fa-reddit-alien fa-2x" title="Reddit">Reddit</i></a></li>
```

Comment:
fa fa-[whatever] is code for 'someone is using Font Awesome' (e.g. https://fontawesome.com/icons/twitter?style=brands). The 'fa' is the generic icon class, 'fa-2x' is twice the default size, and then fa-twitter is the class pointing to the code point in the font file.

I switched fa-users to fa-reddit-alien because it seems more accurate - however, this brings up a second issue (which I messaged you about), which is that someone forgot to actually include Font Awesome as part of the site, so nothing shows up either way.